huaweicloud/services/dli/resource_huaweicloud_dli_flinkjar_job.go

```diff
@@ -366,34 +377,40 @@ func resourceFlinkJarJobDelete(_ context.Context, d *schema.ResourceData, meta i
 func resourceFlinkJarJobUpdate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
 	config := meta.(*config.Config)
 	region := config.GetRegion(d)
-	client, err := config.DliV1Client(region)
-	if err != nil {
-		return diag.Errorf("error creating DLI v1 client, err=%s", err)
+	if err := updateTagsToResource(config, region, d); err != nil {
+		return diag.FromErr(err)
 	}
 
-	jobId, iErr := strconv.Atoi(d.Id())
-	if iErr != nil {
-		return diag.Errorf("the DLI flink job_id must be number. actual id=%s", d.Id())
-	}
+	if d.HasChangesExcept("tags") {
+		client, err := config.DliV1Client(region)
+		if err != nil {
+			return diag.Errorf("error creating DLI v1 client, err=%s", err)
+		}
 
-	diagErr := authorizeObsBucket(client, d)
-	if diagErr != nil {
-		return diagErr
-	}
+		jobId, iErr := strconv.Atoi(d.Id())
+		if iErr != nil {
+			return diag.Errorf("the DLI flink job_id must be number. actual id=%s", d.Id())
```

Comment:
```suggestion
			return diag.Errorf("the DLI flink job_id must be number, but the actual ID is '%s'", d.Id())
```
SQL job resource ditto

Reply:
fixed